src/Task/RealMeSetupTask.php

```diff
@@ -369,7 +369,7 @@ private function validateRealMeEnvironments($forEnv)
         $allowedEnvs = $this->service->getAllowedRealMeEnvironments();
         if (0 === mb_strlen($forEnv ?? '')) {
             $this->errors[] = _t(
-                RealMeSetupTask::class . '.ERR_ENV_NOT_SPECIFIED',
+                RealMeSetupTask::class . '.ERR_ENV_NOT_SPECIFIED2',
```

Comment:
We changes the value below to reflect CMS 6 sake syntax. We don't want that being used in CMS 5.